reactivesocket-core/src/test/java/io/reactivesocket/frame/LeaseFrameFlyweightTest.java

```diff
@@ -0,0 +1,19 @@
+package io.reactivesocket.frame;
+
+import org.agrona.concurrent.UnsafeBuffer;
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+
+import static org.junit.Assert.*;
+
+public class LeaseFrameFlyweightTest {
```

Comment:
For context: the visible change should be that our metadata can be 3 bytes longer in a single frame than previously possible.  But I don't think we cover that with a test, and it isn't clear where it will fail currently when we break that limit? e.g. will it be delayed until the transport? Will transports without frame length encoding silently allow these invalid frames?